### DISCUSSION TITLE
Issue : no matter what i do i always get this

### DISCUSSION THREAD

**Discussion by MrBxcx** [KadeArchive/Kade-Engine#669] [Q&A]
No matter what i do i get this
D:\Kade Mods\Kade-Engine-master>lime test windows
source/Discord.hx:5: characters 8-30 : Type not found : discord_rpc.DiscordRpc
source/Discord.hx:14: characters 3-13 : Type not found : DiscordRpc
source/Discord.hx:24: characters 4-14 : Type not found : DiscordRpc
source/Discord.hx:29: characters 3-13 : Type not found : DiscordRpc
source/Discord.hx:34: characters 3-13 : Type not found : DiscordRpc
source/Discord.hx:39: characters 3-13 : Type not found : DiscordRpc
source/Discord.hx:75: characters 3-13 : Type not found : DiscordRpc
source/Character.hx:502: lines 502-503 : Expected , or )

D:\Kade Mods\Kade-Engine-master>
Someone help me fix this pls

**Comment by isakube** [comment]
install discord_rpc. `haxelib git discord_rpc https://github.com/Aidan63/linc_discord-rpc`

**Comment by MrBxcx** [comment]
i already did.

**Comment by MrBxcx** [comment]
and that stuff still happens
